rustfst/src/semirings/tropical_weight.rs

```diff
@@ -150,9 +147,7 @@ impl SerializableSemiring for TropicalWeight {
     }
 
     fn parse_text(i: &str) -> IResult<&str, Self> {
```

Comment:
I checked and saw that https://github.com/rust-bakery/nom/pull/1673 is merged, so I cleaned this up.

Reply:
Thanks !

Reply:
CI seems to be failing because of that, so I would revert this one, I believe.

Reply:
I can remove this sure. I find the test a bit hard to read. Where can I actually see the input? Was it really `inity,Infinity`? I assume it parsed an `Inf` and left the rest. Should I follow up with nom, as well?

```
---- tests_openfst::test_weights::test_openfst_weight_003 stdout ----
thread 'tests_openfst::test_weights::test_openfst_weight_003' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Error { input: "inity,Infinity", code: Tag })', rustfst/src/tests_openfst/test_weights.rs:37:53
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

https://github.com/Garvys/rustfst/actions/runs/7880821432/job/21718406380?pr=253#step:6:1798

Reply:
I've added it back with a minor update to the comment.

Reply:
Perfect thanks !

Reply:
Honestly I would need to dig to understand exactly what is happening and how this `inity` got there. I always found nom errror tobe very hard to read